src/plugins/console/public/lib/mappings/mappings.js

```diff
@@ -250,7 +250,10 @@ function retrieveSettings(settingsKey, settingsToRetrieve) {
 
   // Fetch autocomplete info if setting is set to true, and if user has made changes.
   if (settingsToRetrieve[settingsKey] === true) {
-    return es.send('GET', settingKeyToPathMap[settingsKey], null, true);
+    // Appending pretty=false here to compress the response size for autocomplete.
```

Comment:
nit: I think we can be a bit more explicit with the comment, what do you think about? `Use pretty=false in these request in order to compress the response by removing whitespace`